quasar-utils/src/main/groovy/net/corda/plugins/QuasarPlugin.groovy

```diff
@@ -36,11 +36,11 @@ class QuasarPlugin implements Plugin<Project> {
         project.dependencies.add("compileOnly", quasar)
 
         project.tasks.withType(Test) {
-            jvmArgs "-javaagent:${project.configurations.quasar.singleFile}"
+            jvmArgs "-javaagent:${project.configurations.quasar.singleFile}=x(antlr**;bftsmart**;co.paralleluniverse**;com.codahale**;com.esotericsoftware**;com.fasterxml**;com.google**;com.ibm**;com.intellij**;com.jcabi**;com.nhaarman**;com.opengamma**;com.typesafe**;com.zaxxer**;de.javakaffee**;groovy**;groovyjarjarantlr**;groovyjarjarasm**;io.atomix**;io.github**;io.netty**;jdk**;junit**;kotlin**;net.bytebuddy**;net.i2p**;org.apache**;org.assertj**;org.bouncycastle**;org.codehaus**;org.crsh**;org.dom4j**;org.fusesource**;org.h2**;org.hamcrest**;org.hibernate**;org.jboss**;org.jcp**;org.joda**;org.junit**;org.mockito**;org.objectweb**;org.objenesis**;org.slf4j**;org.w3c**;org.xml**;org.yaml**;reflectasm**;rx**;org.jolokia**)"
```

Comment:
These exclusions are just an optimisation for running the tests, but the problem is that this kind of information should be configurable instead of hard-coded into the plugin. So (for example), the `quasar-utils` plugin would register a Gradle extension with a `List<String>` property and then a user could populate that property with package-exclusion patterns. The plugin would then "map" those patterns into a correctly-formatted `jvmArgs` value for all `Test` and `JavaExec` tasks.

The issue with hard-coded configuration data is that someone will _always_ want to modify it, and that then requires changing the code.

Reply:
Unfortunately I have no experience with gradle/groovy. After looking into it for a while, this is what I came up with (the new commit). I hope this meets your expectations.

Reply:
Actually this is good, although not _quite_ what I meant :wink:. I think we should also handle the case where `quasar_extensions` is empty, in which case there wouldn't be anything to append to the `-javaagent` parameter.

Reply:
Well I'm glad it's good :)
I added a check for an empty list of exclusions.